dot/sync/chain_processor_test.go

```diff
@@ -375,67 +363,7 @@ func Test_chainProcessor_processBlockData(t *testing.T) {
 					blockState: mockBlockState,
 				}
 			},
-			blockData:     &types.BlockData{},
-			expectedError: mockError,
-		},
-		"handle AddBlockToBlockTree error": {
-			chainProcessorBuilder: func(ctrl *gomock.Controller) chainProcessor {
-				mockBlock := &types.Block{
-					Header: types.Header{
-						Number: uint(1),
-					},
```

Comment:
Note these few test cases are now unneeded for this function and moved to the new tests testing sub-functions.